website/blog/how-to-migrate-an-ios-app-to-bazel.md

```diff
@@ -0,0 +1,1470 @@
+
+An [`apple_intent_library`][apple_intent_library] target generates source files for an `.intentdefinition` file.
+Use it if you have `.intentdefinition` resources.
+
+:::note
+
+The [`swift_intent_library`][swift_intent_library] and [`objc_intent_library`][objc_intent_library] macros wrap `apple_intent_library` with [`swift_library`](#swift_library) and [`objc_library`](#objc_library) targets respectively.
+Use them instead of `apple_intent_library` directly if possible.
+
+[objc_intent_library]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-resources.md#objc_intent_library
+[swift_intent_library]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-resources.md#swift_intent_library
+
+:::
+
+[apple_intent_library]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-resources.md#apple_intent_library
+
+#### `apple_resource_bundle`
+
+An [`apple_resource_bundle`][apple_resource_bundle] target generates a resource bundle.
+Use it if you require certain resources to be placed in a named `.bundle`,
+instead of directly placed in your top level bundle
+(e.g. an `.app`, `.framework` , or `.xctest`).
+
+:::note
+
+`apple_resource_bundle` targets need to be listed in `data` attributes,
+not `deps` attributes.
+
+:::
+
+[apple_resource_bundle]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-resources.md#apple_resource_bundle
+
+#### `ios_application`
+
+An [`ios_application`][ios_application] target generates an application bundle.
+
+Unlike Xcode,
+Bazel separates compilation and bundling targets.
+This means that an `ios_application` target
+(which is a bundling target)
+doesn't list its source files,
+and instead requires that its primary module be its single [static library](#swift_library) dependency
+(which is a compilation target).
+
+[ios_application]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-ios.md#ios_application
+
+#### `ios_app_clip`
+
+An [`ios_app_clip`][ios_app_clip] target generates an app clip bundle.
+
+`ios_app_clip` is used nearly identically to [`ios_application`](#ios_application)
+(since app clips are on-demand applications),
+except that it also needs to be listed in a parent `ios_application`'s `app_clip` attribute.
+
+[ios_app_clip]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-ios.md#ios_app_clip
+
+#### `ios_extension`
+
+An [`ios_extension`][ios_extension] target generates an application extension bundle.
+
+Similar to [`ios_application`](#ios_application),
+`ios_extension` defines a bundling target,
+which means that it doesn't list its source files,
+and instead requires that its primary module be its single [static library](#swift_library) dependency.
+
+:::note
+
+Extensions are listed in the [`extensions`][extensions-attr] attribute of the containing application instead of the `deps` attribute.
+
+[extensions-attr]: https://github.com/bazelbuild/rules_apple/blob/3.3.0/doc/rules-ios.md#ios_application-extension
```

Comment:
This link doesn't resolve to anything for me in Safari, that I can tell. Just the top of the doc.